clients.md

```diff
@@ -23,7 +23,7 @@ The filtering can be done either using the OPTIMADE filter language (see the spe
 
 ## optimade.science
 
-An [open-source](https://github.com/tilde-lab/optimade.science){:target="_blank"} web client developed by [Evgeny Blokhin](https://tilde.pro){:target="_blank"} utilizing [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS){:target="_blank"} is available.
-It allows for searching through *multiple* OPTIMADE databases using the OPTIMADE filter language (see the specification for more information) and the power of CORS.
+An [open-source](https://github.com/tilde-lab/optimade.science){:target="_blank"} web client developed by [Tilde MI](https://tilde.pro){:target="_blank"} utilizing the [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS){:target="_blank"} is available as a single universal `index.html` file. Can be run from the server, USB-stick, etc.
+It allows for searching through *all* the known OPTIMADE databases from the [OPTIMADE providers list](https://providers.optimade.org) *simultaneously* using the OPTIMADE filter language (see the specification for more information).
```

Comment:
While this is *obviously* grating a bit with these *obvious* highlights and comparisons, I think it's fair to highlight the current differences between this and other clients for sure - just wanted to comment on this - no further clarifications or changes are requested from my side.